features/environment.py

```
@@ -1,3 +1,4 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)